frontend/src/components/roundDropdown.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+// This Component is used in the interview page
+// to choose a candidate to add an interview for
+import withRedux from 'next-redux-wrapper'
+import configureStore from './../store/appStore'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import React, { Fragment, Component } from 'react'
+import Select from 'react-select'
+import roundData from '../../../data/roundData.js'
+import { setSelectedRound } from '../actions'
+
+const mapStateToProps = state => ({
+  round: state.round
+})
+
+const mapDispatchToProps = dispatch => {
+  return bindActionCreators(
+    {
+      setSelectedRound
+    },
+    dispatch
+  )
+}
+
+class RoundDropdown extends Component {
+  constructor(props) {
+    super(props)
+  }
+  state = {
+    selectedOption: ''
+  }
+  handleChange = selectedOption => {
+    this.setState({ selectedOption: selectedOption })
+    this.props.setSelectedRound(selectedOption.value)
+  }
+
+  render() {
+    var round_names = []
+    for (var i = 0; i < roundData.numberRounds; i++) {
+      let curr_round = roundData.rounds[i]
+      let curr_round_label = {
+        value: i,
+        label: curr_round.name.toString()
+      }
+      round_names.push(curr_round_label)
+    }
```

Comment:
This is good, no need for change. You could also use `map` here for simplicity but it's just a matter of preference! `map` would be very useful for looping through arrays and generating JSX elements in `render()` though!
```Javascript
roundData.map((round,idx) => {value: idx, label: round.name.toString()})